swagger-java-wes-client/src/main/resources/workflow-execution_service-1.0.0-oas3-swagger.yaml

```diff
@@ -0,0 +1,688 @@
+openapi: 3.0.0
```

Comment:
If this is checked into our repo, please provide instructions on how this was converted from https://github.com/ga4gh/workflow-execution-service-schemas/blob/master/openapi/workflow_execution_service.swagger.yaml

Reply:
https://ucsc-cgl.atlassian.net/browse/DOCK-573